<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QPushButton" name="btTry">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>610</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>试用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btReg">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>610</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btMain">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>460</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>进入主界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbFig">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>661</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lbFig</zorder>
    <zorder>btTry</zorder>
    <zorder>btReg</zorder>
    <zorder>btMain</zorder>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>机器码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ldUuid">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册码</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tdRegCode">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>300</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btReg2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>480</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请向管理员处提供机器码以对设备进行注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btRegBk">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>480</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>620</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>到期时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbLastDate">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>614</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QProgressBar" name="pgSaver">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>630</y>
       <width>116</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>630</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>保存进度</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>540</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btOpenDir">
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btCalcu">
        <property name="text">
         <string>开始解算</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btStop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>1021</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayout"/>
    </widget>
    <widget class="QListWidget" name="lwFileList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>441</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="lbStatus">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>630</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>计算进度</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>440</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>检测阈值</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>宽度门限</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ldThresHold">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ldWideThold">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>计算采样率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ldSampleRate">
        <property name="text">
         <string>48000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>390</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btCheckFile">
        <property name="text">
         <string>查看数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btPrev">
        <property name="text">
         <string>上一帧</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btNext">
        <property name="text">
         <string>下一帧</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lbStatus_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>620</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>630</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QProgressBar" name="pbCurlist">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pbCurfile">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>630</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据帧</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>450</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbShow">
        <property name="text">
         <string>数据回显</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSave">
        <property name="text">
         <string>保存时频片段</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbBatchExec">
        <property name="text">
         <string>批处理</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lbFrame">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>630</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
